release/models/bgp/openconfig-bgp-errors.yang

```diff
@@ -18,7 +18,13 @@ submodule openconfig-bgp-errors {
     "This module defines BGP NOTIFICATION message error codes
     and subcodes";
 
-  oc-ext:openconfig-version "5.4.0";
+  oc-ext:openconfig-version "5.5.0";
+
+  revision "2023-06-05" {
+    description
+      "Add link bandwidth extended community type";
```

Comment:
(nit) full stop on the end of this sentence.

Reply:
reverted unintended changes to bgp